### ISSUE TITLE
feature request - upload direct into "add labels" or "unified view"

### ISSUE THREAD

**Issue by dobkeratops** [bbernhard/imagemonkey-core#255]
Imagine if you could drop an image directly into the add-labels or unified view - to annotate immiediately,
This would be useful for setting up examples - e.,g. when you found an image with unusual,good object examples and it's fresh in your mind

Alternative ways of acheiving the same result:-

- a shortcut in the "donate" page to go into add labels mode (using the most recent upload)
- a search mode in any of the browse modes to see the most recent image uploads, or most recent unlabelled examples (this would be the best)

usually the best default would be a random ordering, but there's the use case of having a specific unusual example fresh in your mind - finding examples for new labels

**Comment by bbernhard** [comment]
> a search mode in any of the browse modes to see the most recent image uploads, or most recent unlabelled examples (this would be the best)

agreed, I think that's the most generic solution to the problem :) 

I'll look into that next. If it's not too much work, I'll add it right away.
